packages/skeleton-react/vite.config.ts

```diff
@@ -13,6 +13,11 @@ export default defineConfig({
 		RemixRouter(),
 		skeletonPluginWatcher(path.resolve(path.join('..', '..', 'packages', 'skeleton', 'src', 'plugin')))
 	],
+	resolve: {
+		alias: {
+			'@': path.resolve(__dirname, './src')
```

Comment:
In Svelte land we always use `$lib` as the alias, if it's possible could we keep the same alias across our component libraries? I'm not sure if `@` is a react convention but if not we should probably aim to keep as much parity between the codebases.

Reply:
I can definitely do that either before I go to work this evening, or after I get home tomorrow morning. Should be a quick, easy commit.

Reply:
Yeah, I don't care what the alias exactly is but consistency between projects would be nice both for us and future contributors